src/core_plugins/kibana/public/management/sections/indices/_create.html

```diff
@@ -38,18 +34,18 @@
               <input
                 ng-model="index.isTimeBased"
                 type="checkbox">
-              Index contains time-based events
+              <span translate="KIBANA-CONTAINS_TIME_BASED_EVENTS_CHECKBOX"></span>
             </label>
           </div>
 
           <div class="form-group" ng-if="index.isTimeBased">
             <label>
-              Time-field name
+              <span translate="KIBANA-TIME_FIELD_NAME"></span>
               &nbsp;
               <kbn-info info="This field will be used to filter events with the global time filter"></kbn-info>
```

Comment:
Hmm yeah I see what you're saying, I guess we'll have to live with the filter for now.